### PR TITLE
[feat][refactor][chore] Create Card Documentation Site

### PR DESCRIPTION
**Pull request by Hamdrive**
1. Created Card documentation site
2. Refactored cards as per requirement
3. Cleaned up irrelevant code

card-doc (1) (1)

### CODE REVIEW

File: Documentation/css/card.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+.example {
+  width: 100%;
+  margin: 1rem 0;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 16px;
+  padding: 16px 0;
```
Comment (stutisk) [inline Documentation/css/card.css]: Use rem instead of px

**Review by stutisk — COMMENTED** [review]
Everything else  looks good to me

File: Documentation/css/card.css (hunk 1 above)
Comment (Hamdrive) [reply Documentation/css/card.css]: Noted!

**Review by AnuragSahu11 — COMMENTED** [review]
Keep up the good work

**Comment by Hamdrive** [comment]
Merging to dev branch

**Pull request merged**